### PR TITLE
Add Somfy deprecation notice

### PR DESCRIPTION
**Pull request by iMicknl**
## Proposed change
See https://github.com/home-assistant/core/issues/69558 for more context. It would be good to clarify that Somfy is sunsetting their current API and that new users should move to Overkiz already. Overkiz integration is fully functional and will support the new local API as well.

Unfortunately, due to different auth methods we cannot migrate users. I will work on adding a deprecation notice to core as well.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by tetienne — DISMISSED** [review]

**Comment by iMicknl** [comment]
image

It seems that the div does not support markdown links. Should I create these links in HTML, or am I doing something else wrong?

**Comment by klaasnicolaas** [comment]
https://developers.home-assistant.io/docs/documenting/create-page#html 😉

**Comment by iMicknl** [comment]
image

Looks better! Thanks for pointing me to the docs. Oops 🙄

File: source/_integrations/somfy.markdown [hunk 1]
```diff
@@ -17,6 +17,12 @@ ha_platforms:
   - switch
 ---
 
+<div class="note warning">
+
+The Somfy Open API (cloud-based) has been deprecated in favour of the [Somfy TaHoma Developer Mode](https://developer.somfy.com/developer-mode) and will shutdown after June 21st 2022. The [Overkiz integration](/integrations/overkiz/) will support this new local API and can already be used to control your Somfy devices via the cloud.
```
Comment (frenck) [inline source/_integrations/somfy.markdown]:
```suggestion
The Somfy Open API (cloud-based) has been deprecated in favor of the [Somfy TaHoma Developer Mode](https://developer.somfy.com/developer-mode) and will shut down after June 21st, 2022. The [Overkiz integration](/integrations/overkiz/) will support this new local API and can already be used to control your Somfy devices via the cloud.
```

**Review by frenck — APPROVED** [review]
Thanks, @iMicknl 👍

**Pull request merged**